packages/team-management/src/components/TeamsView/index.tsx

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Button, Input, notification } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import TeamsDetail, { TeamsDetailProps } from '../TeamsDetail';
+import { SearchOutlined } from '@ant-design/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import reducer, {
+  fetchOrganizationsAction,
+  getOrganizationsArray,
+  Organization,
+  reducerName,
+} from '../../ducks/organizations';
+import { getAccessToken } from '@onaio/session-reducer';
+import { API_BASE_URL, TEAMS_ALL, TEAM_PRACTITIONERS, URL_ADD_TEAM } from '../../constants';
+import Table, { TableData } from './Table';
+import './TeamsView.css';
+import { Spin } from 'antd';
+import { Link } from 'react-router-dom';
+import { Practitioner } from '../../ducks/practitioners';
+
+reducerRegistry.register(reducerName, reducer);
```

Comment:
Let's add some doc strings here

Reply:
@kelvin-muchiri Does this reducerRegistry needs docstring ? 
Because this import from a package.

Reply:
@haseemVD The most common docstring being used across the repo for this statement is `/** Register reducer */`. We can also add that here so that a new person viewing the code will know what that statement does instead of leaving it empty